h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { describe, expect, it } from "vitest";
+import { render } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import * as React from "react";
+import { Breadcrumbs, BreadcrumbsProps } from "./Breadcrumbs";
+
+describe(Breadcrumbs.name, () => {
+  const template = ({ breadcrumbs }: BreadcrumbsProps): HTMLElement => {
+    const { container } = render(
+      <MemoryRouter>
+        <Breadcrumbs breadcrumbs={breadcrumbs} />,
+      </MemoryRouter>,
+    );
+    return container;
+  };
+
+  it("No breadcrumbs should render nothing", () => {
```

Comment:
Test names should make sense to read after "is", e.g:

```suggestion
  it("should render no links if the list of breadcrumbs is empty", () => {
```